src/containers/WorkspaceFullScreenButton.js

```diff
@@ -22,20 +22,8 @@ const mapStateToProps = state => ({
  */
 const mapDispatchToProps = { setWorkspaceFullscreen: actions.setWorkspaceFullscreen };
 
-/**
```

Comment:
It seems like some of these styles were important to keeping the toolbar layout appearing right; it's possible the work in #2225 addressed this though?